ossapi/ossapiv2.py

```diff
@@ -2900,6 +2901,50 @@ def user_recent_activity(
         params = {"limit": limit, "offset": offset}
         return self._get(list[_Event], f"/users/{user_id}/recent_activity/", params)
 
+    @request(Scope.PUBLIC, category="users")
+    def search_beatmaps_passed(
+            self,
+            user_id: UserIdT,
+            *,
+            beatmapset_ids: list[BeatmapsetIdT] = [],
```

Comment:
(I do actually want None default here, partly because mutable default arguments in python [is a footgun](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments))

Reply:
Fair enough - tbh I don't know why it's even optional in the API to begin with...